ctt/uploaders.py

```diff
@@ -180,6 +179,20 @@ def process(self, processing_job, target_as_source=False):
         )
 
 
+class ExtraTagUploader:
+    '''
+    Uploader that will push additional (static) tags to uploaded images. Useful to e.g. add
+    `latest` tag. Extra-Tags will be overwritten as a hardcoded behaviour of this uploader.
+    '''
+    def __init__(self, extra_tags: list[str]):
+        self.extra_tags = tuple(extra_tags)
```

Comment:
Why requiring `extra_tags` to be a list just to cast it to a tuple?

Reply:
yeah, I was lazy. actually it just needs to be typing.Iterable. -> will update